webui/react/src/hooks/usePolling.ts

```diff
@@ -47,9 +51,9 @@ const usePolling = (pollingFn: PollingFn, options: PollingOptions = {}): Polling
     poll();
   }, [ poll ]);
 
-  const stopPolling = useCallback(() => {
+  const stopPolling = useCallback((options: StopOptions = {}) => {
```

Comment:
unrelated to this pr: could we just   return `stopPolling` instead of ` return () => stopPolling();` line 67

Reply:
I used to return the function itself (if it's the only thing that needs to run during cleanup) but I ran into issues of the cleanup not running when it's returned in that format. 🤷